### PR TITLE
Add viewport resize behavior post

### LINKED ISSUE #4003: [CSS] Prepare for viewport resize behavior changes coming to Chrome on Android
* Proposed title: Prepare for viewport resize behavior changes coming to Chrome on Android
* Proposed by: bramus
* Related areas: CSS
* Content type: News/Update (example: https://web.dev/open-web-docs/)
* [Draft](https://docs.google.com/document/d/1zKXpJNP2OM6gHjtdZqmIQcbPPLOsmRWKhvDoHYdTzh0/edit)
* Content summary: What changes are coming to the viewport resize behavior in Chrome 108, why Chrome is making this change, and what you can do to prepare.

### PR DESCRIPTION
**Pull request by bramus**
Fixes #4003

### CODE REVIEW

File: site/en/blog/viewport-resize-behavior/index.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+### Side-effects of each behavior
+
+This difference in how the various viewports get resized when the OSK is shown leads to a non-interoperable layout and sizing behavior of websites.
+
+In the browsers from [group 1](#group-one), with the OSK shown:
+
+  - The computed values for viewport-relative units remain the same
+  - Elements that were designed to take up the full visual space keep their size
+  - Elements that use `position: fixed` remain in place and can be obscured by the OSK
```
Comment (rachelandrew) [inline site/en/blog/viewport-resize-behavior/index.md:133]:
```suggestion
  - The computed values for viewport-relative units remain the same.
  - Elements that were designed to take up the full visual space keep their size.
  - Elements that use `position: fixed` remain in place and can be obscured by the OSK.
```

File: site/en/blog/viewport-resize-behavior/index.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+In the browsers from [group 2](#group-two), with the OSK shown:
+
+  - The computed values for viewport-relative units shrink
+  - Elements that were designed to take up the full visual space shrink
+  - Elements that use `position: fixed` can end up elsewhere in the layout
```
Comment (rachelandrew) [inline site/en/blog/viewport-resize-behavior/index.md:139]:
```suggestion
  - The computed values for viewport-relative units shrink.
  - Elements that were designed to take up the full visual space shrink.
  - Elements that use `position: fixed` can end up elsewhere in the layout.
```

File: site/en/blog/viewport-resize-behavior/index.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+<figure>
+  {% Img src="image/AeNB0cHNDkYPUYzDuv8gInYA9rY2/6ImZDFRlY4pyQoxELETG.png", alt="Visualization of the side-effects in both groups. Note the different position for the elements that use position: fixed (blue boxes)", width="800", height="450", class="screenshot" %}
```
Comment (rachelandrew) [inline site/en/blog/viewport-resize-behavior/index.md:142]:
```suggestion
  {% Img src="image/AeNB0cHNDkYPUYzDuv8gInYA9rY2/6ImZDFRlY4pyQoxELETG.png", alt="Visualization of the side-effects in both groups. Note the different position for the elements that use position: fixed (blue boxes).", width="800", height="450", class="screenshot" %}
```

File: site/en/blog/viewport-resize-behavior/index.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+```html
+<meta name="viewport" content="width=device-width, initial-scale=1.0, interactive-widget-content=overlays">
+```
+
+Accepted values for `interactive-widget` are:
+
+- `resizes-visual` = Resize only the Visual Viewport but not the Layout Viewport
+- `resizes-content` = Resize both the Visual Viewport and Layout Viewport
+- `overlays-content` = Do not resize any viewport
```
Comment (rachelandrew) [inline site/en/blog/viewport-resize-behavior/index.md:172]:
```suggestion
- `resizes-visual`: Resize only the Visual Viewport but not the Layout Viewport.
- `resizes-content`: Resize both the Visual Viewport and Layout Viewport.
- `overlays-content`: Do not resize any viewport.
```

**Review by rachelandrew — APPROVED** [review]
Just some minor formatting to look at before merging.

**Comment by bramus** [comment]
Thank you, @rachelandrew.

File: site/en/blog/viewport-resize-behavior/index.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
+---
+layout: "layouts/blog-post.njk"
+title: Prepare for viewport resize behavior changes coming to Chrome on Android
+authors:
+  - bramus
+description: >
+  What changes are coming to the viewport resize behavior in Chrome 108, why Chrome is making this change, and what you can do to prepare.
+date: 2022-10-27
```
Comment (rachelandrew) [inline site/en/blog/viewport-resize-behavior/index.md:8]:
```suggestion
date: 2022-10-28
```

**Pull request merged**